test.ps1

```diff
@@ -0,0 +1,17 @@
+param ($testResultDir = ".\TestResults")
+
+$env:LOCAL_TEST_DIR = $testResultDir.Replace("\","/")
+
+Get-ChildItem -Path $testResultDir -Filter *.trx | Remove-Item
+
+docker-compose -f .\docker-compose.build.yml build
+docker-compose -f .\docker-compose.test.yml up
+docker-compose -f .\docker-compose.test.yml down
+
+if ((Get-ChildItem -Path $testResultDir -Filter *.trx | Measure-Object).Count -eq 11) {
+    Write-Host "Test results files present. Please review" -ForegroundColor Green
+    exit 0
+}
+
+Write-Host "Not all test result files are present" -ForegroundColor Red
+exit 1    
```

Comment:
Thinking about this, it might be better suited to a Nuke target? `LocalTest` or similar?

Reply:
Good idea. I will look into it

Reply:
`LocalTest` is neat!
Running from a mac has a couple of issues:
- Missing `LOCAL_TEST_DIR` env. Addressed easily by adding `export LOCAL_TEST_DIR=$TEMP_DIRECTORY` in build.sh

Next one, I'll need some advise. 😄 
```
> [octopusclients_test-centos7 build 4/4] RUN dotnet restore ./source:
 #0 0.301 dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by dotnet)
  #0 0.301 dotnet: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by dotnet)
  [ERR] failed to solve: executor failed running [/bin/sh -c dotnet restore ./source]: exit code: 1 
```

Reply:
I am not sure what is happening with your dotnet here. We could pair up to investigate?

Reply:
This file has been removed, we are not using LOCAL_TEST_DIR any more. Nuke handles paths and locations more efficiently

Reply:
I have made the changes for LOCAL_TEST_DIR. Not sure why you are getting issues with missing Libraries. Maybe we can pair on it?